### ISSUE TITLE
Error of command: import pytie.functions as pytie

### ISSUE THREAD

**Issue by sks175** [raquelsilva/pytie#1]
I added the package using command pip install pytie. But i am getting error message as follow:
import pytie.functions as pytie 
Traceback (most recent call last):

  File "<ipython-input-5-abbb0f792a4c>", line 1, in <module>
    import pytie.functions as pytie

  File "C:\Users\Lenovo\Documents\Python Scripts\pytie.py", line 7, in <module>
    import pytie.functions as pytie

ModuleNotFoundError: No module named 'pytie.functions'; 'pytie' is not a package

**Comment by raquelsilva** [comment]
Sorry for my late reply. I was on vacations. So, Pytie was developed in python2 and you might be using python3. To make it work you need to change the kernel to python2 if you are working with Jupyter notebook or simply call python in your command line if you prefer.

**Comment by dcslagel** [comment]
This issue is a year old, but I tried to reproduce it recently and found that the script name seems to be the issue.

I made a file named `pytie.py` (as in the original issue posting) with the contents:
```
import pytie.functions as pytie
```

Running `python pytie.py` has the output:
```
ModuleNotFoundError: No module named 'pytie.functions'; 'pytie' is not a package
```

However if I then move my `pytie.py` to `mypytie.py`  and run `python mypytie.py, the error is not seen.

Maybe this finding will be useful to anyone else who runs into this error.
DC
